From 63b6ebca485cf076ec40f43fbf8abe64e41b28d3 Mon Sep 17 00:00:00 2001
From: Rob Clark <rob@ti.com>
Date: Mon, 19 Sep 2011 17:11:16 -0500
Subject: [PATCH 1/2] Remove extra DRI2SwapBuffers

Remove duplicate DRI2SwapBuffers from 'A.2 Protocol Requests' section.

Conflicts:

	dri2proto.txt
---
 dri2proto.txt |   21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/dri2proto.txt b/dri2proto.txt
index 9921301..0aef84d 100644
--- a/dri2proto.txt
+++ b/dri2proto.txt
@@ -750,27 +750,6 @@ A.2 Protocol Requests
 	1	8			DRI2 opcode
 	2	8			length
 	4	DRAWABLE		drawable
-      ▶	
-	1	1			Reply
-        1				unused
-	2	CARD16			sequence number
-	4	0			reply length
-	4	CARD32			buffer count
-	4	CARD32			unused
-	4	CARD32			unused
-	4	CARD32			unused
-	4	CARD32			unused
-	4	CARD32			unused
-	4	CARD32			unused
-	5n	LISTofDRI2BUFFER	buffers
-└───
-
-┌───
-    DRI2SwapBuffers
-	1	CARD8			major opcode
-	1	8			DRI2 opcode
-	2	8			length
-	4	DRAWABLE		drawable
 	4	CARD32			target_msc_hi
 	4	CARD32			target_msc_lo
 	4	CARD32			divisor_hi
-- 
1.7.9.5

